ext/handlers/filters/message/message.go

```diff
@@ -47,6 +49,18 @@ func ForwardFromChatID(id int64) filters.Message {
 	}
 }
 
+func Regex(p string) filters.Message {
+	return func(m *gotgbot.Message) bool {
+		r, err := regexp.Compile(p)
+		if err != nil {
+			log.Fatal(err)
+			return false
+		}
+		matched := r.MatchString(m.Text)
+		return matched
```

Comment:
```suggestion
	  return r.MatchString(m.Text)
```

No point having a temporary variable here